### DISCUSSION TITLE
point_0696

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#248] [Roy]
"20190523T104031_20190523T104026_T32ULB"
![image](https://user-images.githubusercontent.com/54947195/147513241-b822aaad-cabb-4a40-a161-85b5b1fc938e.png)

"20180826T104021_20180826T104155_T31UGS"
![image](https://user-images.githubusercontent.com/54947195/147513352-8af6acff-7348-49c7-ab83-8682ac94e9d8.png)
![image](https://user-images.githubusercontent.com/54947195/147513466-8d72fd81-8d0d-4af4-ba06-780fe7b5f334.png)

"20180209T103149_20180209T103152_T32ULB"
![image](https://user-images.githubusercontent.com/54947195/147513733-24f520c4-81ca-4683-829f-51137bd7d024.png)
Aquí en esta imagen  le pondría  4 de dificultad 

@ryali93

**Comment by AntoniCesar** [comment]
"20190523T104031_20190523T104026_T32ULB"
![Captura](https://user-images.githubusercontent.com/86502862/152885566-0dcbcb04-2e3b-4860-a1b9-3cfff029efdc.JPG)

**Comment by AntoniCesar** [comment]
"20180826T104021_20180826T10
![Captura](https://user-images.githubusercontent.com/86502862/152889360-3590487e-ebdd-422c-8b22-a0416f90f3e1.JPG)
4155_T31UGS"
